src/main/java/com/google/ehub/servlets/UserProfile.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+public class UserProfile {
+  private final String username;
+  private final String bio;
+  private final String email;
+  private final String name;
+
+  public UserProfile(String name, String username, String bio, String email) {
```

Comment:
You could make this class final since there is no reason for us to extend it within the scope of our project.

Reply:
That makes sense, I will make it final as well.